prime-dotnet-webapi/Migrations/20200911200512_AddedSiteStatus.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Prime.Migrations
+{
+    public partial class AddedSiteStatus : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "Status",
+                table: "Site",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.CreateTable(
+                name: "NewestAgreements",
+                columns: table => new
+                {
+                    Id = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_NewestAgreements", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "SiteRegistrationNote",
```

Comment:
I looks like this migration was made off a stale ApiDbContextModelSnapshot, it doesn't include changes that Nathan made